core/scss/utilities/mixins/typography/_small-paragraph.scss

```diff
@@ -0,0 +1,12 @@
+@charset "UTF-8";
+
+//
+// @small-paragraph
+//
+// To be used for describing the content of an image.
+//
+// Style guide: Mixins.Typography.small-paragraph
+//
+@mixin small-paragraph {
+  font-size: 1.8rem;
+}
```

Comment:
```suggestion
 @include grid-media-max('sm') {
  font-size: 1.5rem;
 }
}
```

Reply:
Looking at Figma and it seems like at xs and sm, the font-size is 15px.

Reply:
Thanks for adding the responsiveness for this!